handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/HystrixCommandPropertiesExt.java

```diff
@@ -178,11 +178,11 @@
   private final HystrixProperty<Boolean> requestCacheEnabled;
 
   protected HystrixCommandPropertiesExt(HystrixCommandKey key) {
-    this(key, HystrixCommandProperties.Setter(), "cse");
+    this(key, HystrixCommandProperties.Setter(), "servicecomb");
```

Comment:
we need to add a test to test this. And we need to test if configure cse.xxx work.

Reply:
i'll add some test later